src/sonic-config-engine/minigraph.py

```diff
@@ -209,12 +209,22 @@ def parse_dpg(dpg, hname):
                 # This ACL has no interfaces to attach to -- consider this a control plane ACL
                 try:
                     aclservice = aclintf.find(str(QName(ns, "Type"))).text
-                    acls[aclname] = {'policy_desc': aclname,
-                                     'ports': acl_intfs,
-                                     'type': 'CTRLPLANE',
-                                     'service': aclservice if aclservice is not None else 'UNKNOWN'}
+
```

Comment:
now, we are looking attachto to decide whether it is dataplane or control plane acl, we need to make sure that the attachto interface is really front panel ports. We should ignore loopback or management in that attachto.